_posts/2023-07-05-trino-fest-2023-arcticwolf.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "AWS Athena (Trino) in the cybersecurity space"
+author: "Anas Shakra, Ryan Duan"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/ArcticWolf.png
+show_pagenav: false
+---
+
+AWS Athena has become a new and efficient service for companies to query their
+data using Trino. AWS Athena is a serverless, interactive analytics service
+built on open-source frameworks, supporting open table and file formats and
+providing a simplified, flexible way to analyze petabytes of data where it lives.
+Arctic Wolf Networks, a cybersecurity company that provides security monitoring
```

Comment:
I would re-order this paragraph. Start by introducing who Arctic Wolf is (it's their talk), then explain what Athena is after mentioning that htey use it. You should also be more explicit about the fact that Athena runs on Trino.